src/test/java/uk/sky/cqlmigrate/CassandraNoOpLockingMechanismTest.java

```diff
@@ -1,101 +1,22 @@
-import org.scassandra.junit.ScassandraServerRule;
-import uk.sky.cqlmigrate.util.PortScavenger;
-
-import java.util.UUID;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.scassandra.http.client.PrimingRequest.then;
-import static org.scassandra.http.client.types.ColumnMetadata.column;
 
+@RunWith(MockitoJUnitRunner.class)
 public class CassandraNoOpLockingMechanismTest {
 
-    private static final int BINARY_PORT = PortScavenger.getFreePort();
-    private static final int ADMIN_PORT = PortScavenger.getFreePort();
-    private static final String CLIENT_ID = UUID.randomUUID().toString();
-
-    @ClassRule
-    public static final ScassandraServerRule SCASSANDRA = new ScassandraServerRule(BINARY_PORT, ADMIN_PORT);
-
-    @Rule
-    public final ScassandraServerRule resetScassandra = SCASSANDRA;
-
-    private final PrimingClient primingClient = SCASSANDRA.primingClient();
-    private final ActivityClient activityClient = SCASSANDRA.activityClient();
-
-    private Cluster cluster;
-    private CassandraNoOpLockingMechanism lockingMechanism;
-
-    @Before
-    public void baseSetup() throws Exception {
-
-        cluster = Cluster.builder()
-                .addContactPoint("localhost")
-                .withPort(BINARY_PORT)
-                .build();
+    public static final String CLIENT_ID = "CLIENT_ID";
 
-        primingClient.prime(PrimingRequest.preparedStatementBuilder()
-                .withQuery("INSERT INTO cqlmigrate.locks (name, client) VALUES (?, ?) IF NOT EXISTS")
-                .withThen(then()
-                        .withVariableTypes(PrimitiveType.TEXT, PrimitiveType.TEXT)
-                        .withColumnTypes(column("[applied]", PrimitiveType.BOOLEAN))
-                        .withRows(ImmutableMap.of("[applied]", true)))
-                .build()
-        );
-
-        primingClient.prime(PrimingRequest.preparedStatementBuilder()
-                .withQuery("DELETE FROM cqlmigrate.locks WHERE name = ? IF client = ?")
-                .withThen(then()
-                        .withVariableTypes(PrimitiveType.TEXT, PrimitiveType.TEXT)
-                        .withColumnTypes(column("[applied]", PrimitiveType.BOOLEAN))
-                        .withRows(ImmutableMap.of("[applied]", true)))
-                .build()
-        );
-
-        lockingMechanism = new CassandraNoOpLockingMechanism();
-        lockingMechanism.init();
-
-        activityClient.clearAllRecordedActivity();
-    }
-
-    @After
-    public void baseTearDown() throws Exception {
-        cluster.close();
-    }
+    private final CassandraNoOpLockingMechanism lockingMechanism = new CassandraNoOpLockingMechanism();
 
     @Test
-    public void shouldPrepareNoInsertLocksQueryWhenInit() throws Throwable {
+    public void shouldDoNothingWhenInit() throws Throwable {
         //when
         lockingMechanism.init();
```

Comment:
I assume this test can go away and be moved to a constructor? The init needs to be called for consecutive steps.